models/fixtures/user.yml

```diff
@@ -218,3 +218,33 @@
   avatar_email: user13@example.com
   num_repos: 3
   is_active: true
+
```

Comment:
Oh, I missed that user15 and user16 had the same email addresses; sorry about that.

Instead of introducing two new fixtures for testing the duplicate email case, could we instead do something like:
```
func TestDuplicateEmailFailedLogin(t *testing.T) {
	prepareTestEnv(t)
	user := models.AssertExistsAndLoadBean(t, &models.User{ID: 2}).(*models.User)

	// add new user with user2's email
	user.Name = "testuser"
	user.LowerName = strings.ToLower(user.Name)
	user.ID = 0 // will be auto-populated on insert
	models.AssertSuccessfulInsert(t, user)
	
	// TODO try to login with user.Email, should fail with duplicate email
}
```

It's not really scalable for us to add new test fixtures for every corner case we need to test.